client/meter/blockchain.js

```diff
@@ -32,9 +32,10 @@ async function mySellerContracts() {
     const seller = await contract.seller();
     if (seller === account) {
       const offeredAmount = await contract.offeredAmount();
-      if (offeredAmount == 0) continue;
+      if (offeredAmount == 0)
+        continue;
       const unitPrice = await contract.unitPrice();
-      contracts.push({contractAddr, unitPrice, offeredAmount})
+      contracts.push({contractAddr: contractAddr, unitPrice: unitPrice, offeredAmount: offeredAmount})
```

Comment:
Why?

Reply:
So that I can access the fields by their name?

Reply:
http://www.benmvp.com/learning-es6-enhanced-object-literals/

Reply:
Oh right, I forgot about those. Will change